pkg/config/pipeline.go

```diff
@@ -117,35 +118,44 @@ func (p *Pipeline) GetSupported() []string {
 
 // ReplacePatterns replaces the phrases that are found in the build file according to
 // the regex pattern with the specified value
-func (p *Pipeline) ReplacePatterns(dir string) []error {
+func (p *Pipeline) ReplacePatterns(config *Config, inputs Replacements, dir string) []error {
 
-	var err error
+	// var err error
```

Comment:
Is this intentionally commented out?

Reply:
Yeah as I was playing with a list of errors. Removed code as now obsolete.